### DISCUSSION TITLE
Adding a new optiong for practicing Keyboard shortcuts

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#2833] [Ideas]
Adding a new section or option for practicing  keyboard shortcut

- Common text editing shortcut like home, ctr + a ,ctr + left or right arrow ...

- Common office purpose tools
  - Microsoft
      - Excel
      - Word
      - Power point
      
  - Google suit
      - Sheet
      - Slide
      - Docs
     
-  Browsers
    - Firefox
    - Chrome

- Desktop environments
    - Windows
    - Mac os
    - Linux
       - Linux mint
       - Gnome
       - i3wm
       - xfc
       - kde
       

 
- Tools 
    - Vim
    - Tmux
    - Terminal ( command linux terminal shortcut like ctrl + a,ctrl + e .. )
    - vs code
    - IntelliJ IDEA
 so on..

**Comment by RenSys** [comment]
Excellent suggestion. One approach would be to mimic [Shortcutfoo](https://www.shortcutfoo.com/) flashcard like system. Users either select from a list of preconfigured application shortcuts or roll their own.
